corehq/messaging/scheduling/util.py

```diff
@@ -0,0 +1,8 @@
+from datetime import datetime
+
+
+def utcnow():
+    """
+    Defined here to do patching in tests.
```

Comment:
I think you could also use something like https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/tests/test_pillow.py#L84